### ISSUE TITLE
Transparent meshes can appear full blue when selected

### ISSUE THREAD

**Issue by Fact87** [Trylz/KronosRenderer#1]
I have the following issue in the real-time preview.
When selecting a transparent object it can appear full blue depending on the camera position. 

Here a screenshot:
![tranparency_issue](https://user-images.githubusercontent.com/40348318/44496616-3eb6ff00-a643-11e8-8bc7-b079b00f0a49.PNG)

**Comment by Trylz** [comment]
Yes the problem is the real-time preview transparency is not order independent.
Will be fixed in the next release.
